robyn/robyn.pyi

```diff
@@ -21,6 +21,10 @@ class Response:
     headers: dict[str, str]
     body: str
 
+    def set_file_path(file_path):
```

Comment:
Can you add `str` typing to the parameter here?

Reply:
@AntoineRR , yep sure 👍

Reply:
done!